centml/values.yaml

```diff
@@ -1,20 +1,31 @@
 nodeSelector:
   region: LGA1
 
+affinity:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+        - matchExpressions:
+            - key: topology.kubernetes.io/region
+              operator: In
+              values:
+                - LGA1
+            - key: gpu.nvidia.com/class
+              operator: In
+              values:
+                - A40
```

Comment:
Most of this should be moved to the template.

To make this easier to use the values file should only have the region (like we do on line 2) and GPU skews we want. The entries in the `nodeSelectorTerms` list should not be in the values file to make it easier to use since we will always want at least a region and GPU class.

Reply:
Moved the block to `centml/templates/notebook-deployment.yaml` and added `gpuClass` in values.yaml